packages/shopify-app-remix/src/server/future/flags.ts

```diff
@@ -0,0 +1,19 @@
+export interface FutureFlags {
+  /**
+   * When enabled, returns the same `admin` context (`AdminApiContext`) from `authenticate.webhook` that is returned from `authenticate.admin`.
+   *
+   * @default false
+   */
+  v3_webhookAdminContext?: boolean;
```

Comment:
Worth showing an example of with and without here?

Reply:
I'll add a link to the actual docs so we don't have to maintain yet another copy of this set of examples!